tests/conftest.py

```diff
@@ -54,15 +20,11 @@ def pytest_generate_tests(metafunc):
 
         response = requests.get(f'{base_url}/menu', auth=(user_obj.username, user_obj.password),
                                 verify=False)
-        if response.status_code == 404:
-            # Menu is only available since Foreman 2.0
-            pages = [base_url + page for page in PAGES]
-        else:
-            assert response
-            pages = [pytest.param(base_url + page['url'], id=page['name'])
-                     for page in response.json()
-                     # logout has an error "Cannot read property 'icon' of null"
-                     if page['url'] != '/users/logout']
+        assert response
+        pages = [pytest.param(base_url + page['url'], id=page['name'])
+                 for page in response.json()
+                 # logout has an error "Cannot read property 'icon' of null"
```

Comment:
Is that still true?

Reply:
I literally removed the line at first and then thought this was already big enough of a change that I didn't want to dig further to see when it was fixed (if at all).